pkgs/development/interpreters/ruby/default.nix

```
@@ -198,6 +198,10 @@ let
     ) args; in self;
 
 in {
+  ruby_build_generic = { version, ... } @ args: generic (args // {
```

```suggestion
  mkRuby = { version, ... } @ args: generic (args // {
```
`mk` or `build` are the name prefixes that are generally used by that produce derivations.

It would be nice is mkRuby could also be used subsequently below to make it clear how to use it.